website/content/docs/release-notes/consul-terraform-sync/v0_6_0_beta.mdx

```diff
@@ -172,3 +174,82 @@ The configuration options are:
 | `agent_pool_id` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;| Optional | string | The execution mode that determines whether to use Terraform Cloud as the Terraform execution platform. Only supports "remote" or "agent". | none |
 | `agent_pool_name` | Optional | string | The name of the agent pool that should run the Terraform workloads. Only supported if `execution_mode` is set to "agent". Either `agent_pool_id` or `agent_pool_name` are required. `agent_pool_id` takes precedence over `agent_pool_name` if both are provided. | none |
 | `terraform_version` &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Optional | string | The version of Terraform to use for the Terraform Cloud workspace associated with the task. Defaults to the latest compatible version supported by the organization. | none |
+
+## Health Endpoint
+The `/health` endpoint returns a successful response if Consul-Terraform-Sync (CTS) is available and running. Requests to this endpoint are not logged, which makes it suitable for health checks that constantly poll CTS.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/health` | `application/json` |
+
+### Response Statuses
+
+| Status | Reason                                               |
+| ------ | ---------------------------------------------------- |
+| 200    | CTS is healthy                                       |
+
+### Response Fields
+
+| Name         | Type   | Description                                                                        |
+| ------------ | ------ | ---------------------------------------------------------------------------------- |
+| `request_id` | string | The ID of the request. Used for auditing and debugging purposes.                   |
+
+### Example
+
+The following request makes a `GET` call to the `health` endpoint:
+
+```shell-session
+$ curl --request GET \
+  localhost:8558/v1/health
+```
+
+Response:
+
+```json
+{
+  "request_id": "b7559ab0-5111-381b-367a-0dfb7e216d41"
+}
+```
+
+## CTS Service Registration With Consul
+CTS by default will automatically register itself as a service in Consul with a health check. The default health check created is an HTTP check to the [health API endpoint](#health-endpoint). If CTS is stopped, it will deregister itself with Consul.
+
+If CTS is unsuccessful at registering itself with Consul, then CTS will log the error and continue without exiting.
+
+~> If ACLs are enabled in Consul, then the [Consul token](/docs/nia/configuration#token) must have the policy `service:write` for service registration to succeed.
+
+### Configuration
+To support registration, CTS now has an `id` configuration that is a global configuration and a `service_registration` configuration that is part of the `consul` configuration. All configurations are optional.
```

Comment:
```suggestion
Specify the `id` parameter to assign instances of CTS an ID. Configure the  `service_registration` block in the `consul` configuration to enable CTS registration. All configurations are optional.
```
Right?

Reply:
```suggestion
Specify the global configuration `id` to assign instances of CTS an ID, which is used as the service ID. Configure the `service_registration` block in the `consul` configuration to configure the registration details. All configurations are optional.
```

I made some modifications, let me know what you think! You don't have to configure `service_registration` to enable it since it's on by default, and I really wanted to keep global configuration since that's how we refer to these top-level configurations ([reference](https://www.consul.io/docs/nia/configuration#global-config-options)).

Reply:
@trujillo-adam Updated it to the following:
> Specify the global configuration `id` to assign instances of CTS an ID, which is used as the service ID for CTS if service registration is enabled. Configure the `service_registration` block in the `consul` configuration to configure the registration details. All configurations are optional.

Please let me know if you have any concerns, I'll be merging this EOD today otherwise so that it's ready for our beta release tomorrow.